app/src/main/contrib/components/Responsibilities/Overview/List/ResponsibilityComponent.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
 export class ResponsibilityComponent extends React.Component<Props, undefined> {
     render() {
         const item = this.props.responsibility;
-        const downloadUrl = `/${ this.props.modellingGroup.id }/responsibilities/${ this.props.touchstone.id }/${ item.scenario.id }/`;
+        const downloadUrl = `/${this.props.modellingGroup.id}/responsibilities/${this.props.touchstone.id}/coverage/${item.scenario.id}/`;
```

Comment:
Now that there is more than just coverage data under the responsibilities list, I had to add an extra layer to the URL hierarchy.